### PR TITLE
Feat/us 33 admin invoice show page

### PR DESCRIPTION
**Pull request by TLamb32**
Add tests and functionality for US 33 and made a few quick fixes in US 32.

### CODE REVIEW

File: app/views/admin/invoices/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h3>Invoice <%=@invoice.id %></h3>
+
+<div id="invoice-details">
```
Comment (Sykogst) [inline app/views/admin/invoices/show.html.erb:3]: I like this, I should implement it in the equivalents on my end to do some within testing

File: spec/features/admin/invoices/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  describe "As an admin, When I visit the admin invoices show page (/admin/invoices/:invoice_id)" do
+    # US 33
+    it "I see information related to that invoice including: id, status, created at, customer first and last name" do
+
+      visit admin_invoice_path(@invoice_1a)
+
+      expect(page).to have_content(@invoice_1a.id)
+      within("#invoice-details") do
+      expect(page).to have_content("Date Created: #{@invoice_1a.formatted_date}")
+      expect(page).to have_content("Customer Name: #{@invoice_1a.customer_name}")
```
Comment (Sykogst) [inline spec/features/admin/invoices/show_spec.rb:17]: Wireframe has it as just Customer:

Comment (TLamb32) [reply spec/features/admin/invoices/show_spec.rb:17]: Will get this fixed in my next PR. Thanks!

File: app/views/admin/invoices/show.html.erb (hunk 1 above)
Comment (TLamb32) [reply app/views/admin/invoices/show.html.erb:3]: Can do some nice nested within blocks as well, if you dare haha

**Pull request merged**